src/setup-locale.js

```diff
@@ -1,28 +0,0 @@
-/**
- * WordPress dependencies
- */
-import { setLocaleData } from '@wordpress/i18n';
-
-export default ( { domain, locale, extraTranslations, getTranslation } ) => {
-	let translations = getTranslation( locale );
-	if ( locale && ! translations ) {
-		// Try stripping out the regional
-		locale = locale.replace( /[-_][A-Za-z]+$/, '' );
-		translations = getTranslation( locale );
-	}
-	const allTranslations = Object.assign(
-		{},
-		translations,
-		extraTranslations
-	);
-	// eslint-disable-next-line no-console
-	console.log(
-		domain ? `${ domain } - locale` : 'locale',
-		locale,
-		allTranslations
-	);
-	// Only change the locale if it's supported by gutenberg
-	if ( translations || extraTranslations ) {
-		setLocaleData( allTranslations, domain );
-	}
-};
```

Comment:
This file is no longer required as locale setup for plugins is now done via the plugin translations param when registering Gutenberg ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/blob/504270493676fe9c2a219519bb70a8d76106dee1/src/index.js#L30)).